portal/lib/appsql.class.php

```diff
@@ -26,6 +26,8 @@
  * @param
  *            wrapper class for moving some care coordination zend product
  */
+$ignoreAuth = true;
+require_once(dirname(__FILE__) . '/../../interface/globals.php');
```

Comment:
hi @sjpadgett  , This makes me very nervous. I think this means that any script that includes this script would be able to skip authentication.

Reply:
I forgot it was there. when you added logger class it was out of scope with auto loader so I meant to go back and include the class direct.

Reply:
doh, sorry about that,
Looks like a couple places that use this script need classes for autoload. We should look at those places to see if either can get global.php included safely or, if not, then just do the autoload by itself.

Reply:
already fixed and will put up in a minute.